<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_PopupDialCorpoFact</class>
 <widget class="QWidget" name="C_PopupDialCorpoFact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>88</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_kg_x">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> mg x</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_QPA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantité de principe actif</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_QPA">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_QFC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>à administrer par (facteur corporel)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_QFC">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>35</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_QPA_by_UP_0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantité de principe actif </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_QPA_by_UP">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>65</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_QPA_by_UP_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>par unité de prise</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_unit_FactCorpo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>poids en kg</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_poids">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>0</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UPA">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>20</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>mg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UFC">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>35</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kg de poids</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
